Sources/Sandbox.Game/Game/Entities/MyEntity.cs

```diff
@@ -100,6 +100,13 @@ public MyGameLogicComponent GameLogic
             set { Components.Add<MyGameLogicComponent>(value); }
```

Comment:
I'm not actually sure what they are doing with components.  I almost feel like it's a Type container for validation or something.  When I did my tests before, it didn't really seem to matter.  I will ask someone in Keen though.